client/src/assets/messages/landing.ts

```diff
@@ -0,0 +1,30 @@
+// Strings for the landing page.
+
+/**
+ * Strings related to the scorecard-related part of the landing page.
+ */
+export class ScorecardSearch {
+  static HEADER = `Get your drinking water score`;
+  static BODY = `Understand the likelihood of your home or community's exposure to lead service lines.`;
+  static CTA_PLACEHOLDER = `Enter your address`;
+  static CTA_BUTTON = 'Get Score';
+}
+
+/**
+ * Strings related to the map-related part of the landing page.
+ */
+export class MapInfo {
+  static HEADER = `Get up to date information about lead service lines`;
+  static BODY = `Stay informed by using the LeadOut Nationwide Map by exploring this ever-growing data set of lead piplines, DEP violations, demographics, and more.`;
+  static CTA_BUTTON = 'Explore Nationwide Map';
+}
+
+/**
+ * Strings related to the filter-related part of the landing page.
+ */
+export class FilterInfo {
+  static SUPER_HEADER = `Take Action`;
+  static HEADER = `Chosing a water filter`;
+  static BODY = `There are several types of water filters. Choosing the right one depends on your budget and lifestyle.`;
+  static CTA_BUTTON = 'Learn more about water filters';
+}
```

Comment:
If there's multiple exports, I've been doing export {FilterInfo, MapInfo, etc..} at the bottom bc it's been helping me to know what's available from every fail at a quick glance

Reply:
I'm don't love that convention (mainly since I'd rather see if something is exported in-line with its definition). The [TS style guide](https://g3doc.corp.google.com/javascript/typescript/g3doc/dev/readability/styleguide.md?cl=head#exports) doesn't seem to have an opinion and CS has plenty of both ([1](https://source.corp.google.com/search?q=lang:ts%20%22export%20%22&sq=), [2](https://source.corp.google.com/search?q=lang:ts%20%22export%20%7B%22&start=11)).

Maybe let do it case-by-case, depending on the size+complexity of a file? This is small enough and exports everything, so it's still easy to see exports at a glance.